js/components/Confirmation.jsx

```diff
@@ -180,27 +182,31 @@ class Confirmation extends React.Component {
                     </div>
                 </button>;
             default:
-                return <button className="confirmation-button" onClick={() => this.resubmitForm()}>Submit</button>;
+                return <button className="confirmation-button" onClick={() => this.submitForm()}>Submit</button>;
         }
     }
 
     render() {
         let confirmationCaseNumber = this.state.confirmationCaseNumber;
         let confirmation_content = null;
 
+        // If the email was already sent (-1), then no need for the form.
         if(this.state.formSubmissionState === -1) {
             console.log("renderConfirmation() The email looks valid, so no need to render form");
-            confirmation_content = <div style={{marginTop: "4rem"}}>
-                You will receive an email with a copy of your complaint and a confirmation number at <b>{this.state.userEmail}</b>
+            confirmation_content = <div className="confirmation-sentacknowledgement">
```

Comment:
not sure that this would follow BEM convention, but barring that atm lets at least seperate the words to it reads `confirmation-sent-acknowledgment`.